### PR TITLE
fix: store orbit meta in separate db

### LINKED ISSUE #99: Truncating OrbitMeta in service provider breaks `shouldRestoreCache` method
The 1.1.0 release adds the following line to the `boot` method of the `OrbitServiceProvider` to truncate the `_orbit_meta` table if it already exists.

https://github.com/ryangjchandler/orbit/pull/95/files#diff-9ab2c0c6607186af00d0a727b04179cd17605199ab1d7662338dd6d7a19ff223R70

Doing so updates the last modified timestamp of the Sqlite database. This introduces a bug in the `shouldRestoreCache` method of the `FileDriver` class since it compares the timestamp of the database file to the last modified timestamp of the content files. This will always return `false` now since the database gets touched at the beginning of each request. This means that the cache never gets updated and we get served stale content until we completely remove the database.

I'm not sure what the best way to fix this is as I didn't look too deeply into the changes in 1.1.0. This might also have slipped through the test suite since the database always gets migrated in tests due to `Orbit::isTesting()` being checked first.

### PR DESCRIPTION
**Pull request by ryangjchandler**
Closes #99.

### CODE REVIEW

**Comment by ryangjchandler** [comment]
@ksassnowski Would you be able to give this branch a go? You should be able to update your `composer.json` to use `dev-fix/orbit-meta-timestamp-problems` as the version.

File: src/OrbitServiceProvider.php [hunk 1]
```diff
@@ -59,8 +66,13 @@ public function boot()
             ], 'orbit:config');
         }
 
-        if (! Schema::connection('orbit')->hasTable('_orbit_meta')) {
-            Schema::connection('orbit')->create('_orbit_meta', function (Blueprint $table) {
+        if (! (new Filesystem())->exists($metaPath = Orbit::getMetaDatabasePath())) {
+            (new Filesystem())->ensureDirectoryExists(dirname($metaPath), 0755, true);
+            (new Filesystem())->put($metaPath, '');
```
Comment (ksassnowski) [inline src/OrbitServiceProvider.php:71]: How about storing the `Filesystem` object in a variable so we don't have to new it up three times?

**Comment by ksassnowski** [comment]
Thanks for the quick fix. I tried out the branch and it's working again as expected :)

**Comment by ryangjchandler** [comment]
Awesome, will merge & tag this shortly.

File: src/OrbitServiceProvider.php (hunk 1 above)
Comment (ryangjchandler) [reply src/OrbitServiceProvider.php]: Just used the `File::` facade instead.

**Pull request merged**